docs/Refund.md

```diff
@@ -0,0 +1,29 @@
+# Refunds
+
+Refunds are one of the use cases that use the `Pull Payments` feature.
+
+Up until now, merchants had to manually issue refunds.
+The process required a lot of back and forth between a sender and a receiver. The merchant had to ask for a refund address, calculate the preferred rate, manually add that to the wallet and broadcast the transaction.
+
+, a merchant just needs to go to the invoice, click Issue refund , select the appropriate rate and share a link with a customer.
+
+The customer will then input their address and claim a refund. Once a customer claims a refund, the merchant will get a notification and can authorize it in a click.
+In future releases, we will look into payouts which are approved to be paid out instantly, without the authorization, in case a merchant prefers that.
```

Comment:
I'd make this more formal. The future versions of btcpay server may.... sth like that.

Reply:
> The future versions of btcpay server may.... sth like that.

Pardon ?

Reply:
Ah sorry, I understand what you were saying now !

Reply:
```suggestion
Future versions of BTCPay Server will enable instant payouts as an option, without the need for merchant approval.
```